docs/install.rst

```diff
@@ -501,8 +500,13 @@ Sherpa was configured when built)::
    and ``--runzenodo`` arguments from the
    :ref:`development build <developer-build>` section.
 
-The ``sherpa`` Anaconda channel contains the ``sherpatest`` package, which
-provides a number of data files in ASCII and :term:`FITS` formats. This is
-only useful when developing Sherpa, since the package is large. It
-will automatically be picked up by the ``sherpa_test`` script
-once it is installed.
+The
+`Sherpa test data suite <https://github.com/sherpa/sherpa-test-data>`_
+contains the ``sherpatest`` package, which provides a number of
+data files in ASCII and :term:`FITS` formats. This is
+only useful when developing Sherpa, since the package is large.
+Example: Installing the 4.14.1 data via pip::
+
+   pip install https://github.com/sherpa/sherpa-test-data/archive/4.14.1.zip
+
+And will then automatically be picked up by the ``sherpa_test`` script.
```

Comment:
```suggestion
A version of the test data is released for each `version of Sherpa <https://doi.org/10.5281/zenodo.593753>`_.

As an example, the 4.15.1 version of the test data can be installed with pip::

   pip install https://github.com/sherpa/sherpa-test-data/archive/4.15.1.zip

The test data will be automatically picked up by the Python tests and the ``sherpa_test`` script.
```

The link I added here isn't quite correct - it doesn't go to a list of releases, just the latest, but
- it does let you find out what the latest release is
- you can get a list of releases from this page

I didn't want to use a URL for the "list all releases" page as it is something that looks like the format could change over time.